src/YQPkgDescriptionView.cc

```diff
@@ -136,7 +136,7 @@ QString YQPkgDescriptionView::simpleHtmlParagraphs( QString text )
     bool foundAuthorsList = false;
     QString html_text = "<p>";
 
-    QStringList lines = text.trimmed().split( '\n', Qt::KeepEmptyParts );
+    const QStringList lines = text.trimmed().split( '\n', Qt::KeepEmptyParts );
```

Comment:
Not sure if this `const` does anything but making the code harder to read. It's not as if the body here would even attempt to modify this variable.

Const references have a use in function parameters: It's not only a promise, but an assurance that the variable will not be modified in the function.

But inside a function, it just adds to the mental burden of reading the code; more information to keep in mind, to think about why this was chosen. It distracts from the code's intent.